eldritch/names.js

```diff
@@ -31,6 +31,7 @@ characterData = {
   "Doctor": {"sliders": [0, 5, 0, 4, 2, 4], "focus": 2, "home": "Hospital"},
   "Archaeologist": {"sliders": [1, 3, 2, 3, 1, 5], "focus": 2, "home": "Shop"},
   "Gangster": {"sliders": [2, 4, 3, 4, 0, 3], "focus": 1, "home": "House"},
+  "Spy": {"sliders": [1,3,2,1,1,0], "focus": 1, "home": "Newspaper"},
```

Comment:
FYI, the UI is not currently representing the sliders properly... Shouldn't be an issue if the art loads, but the default drawing assumes the normal sliders and slider progression.